Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -406,6 +406,64 @@ void ADefiWalletCoreActor::InitializeNewDevelopmentOnlyWallet(
   }
 }
 
+void ADefiWalletCoreActor::GetBackupMnemonicPhrase(FString &output,
+                                                   bool &success,
+                                                   FString &output_message) {
+
+  try {
+    if (NULL == _coreWallet) {
+      success = false;
+      output_message = TEXT("Invalid Wallet");
+      return;
+    }
+
+    assert(_coreWallet != NULL);
+    rust::cxxbridge1::String result = _coreWallet->get_backup_mnemonic_phrase();
+    output = UTF8_TO_TCHAR(result.c_str());
+    success = true;
+  } catch (const rust::cxxbridge1::Error &e) {
+    success = false;
+    output = TEXT("");
+    output_message = FString::Printf(
+        TEXT("CronosPlayUnreal GetBackupMnemonicPhrase Error: %s"),
+        UTF8_TO_TCHAR(e.what()));
+  }
+}
+
+void ADefiWalletCoreActor::DevelopmentOnlyGenerateMnemonics(
+    FString password, EMnemonicsWordCount wordcount, FString &output,
+    bool &success, FString &output_message) {
+  try {
+
+    ::org::defi_wallet_core::MnemonicWordCount mywordcount;
+    switch (wordcount) {
+    case EMnemonicsWordCount::Twelve:
+      mywordcount = ::org::defi_wallet_core::MnemonicWordCount::Twelve;
+      break;
+    case EMnemonicsWordCount::Eighteen:
+      mywordcount = ::org::defi_wallet_core::MnemonicWordCount::Eighteen;
+      break;
+    case EMnemonicsWordCount::TwentyFour:
+      mywordcount = ::org::defi_wallet_core::MnemonicWordCount::TwentyFour;
+      break;
+    default:
+      throw "Invalid Word Count";
+      break;
+    }
+    rust::String result =
+        generate_mnemonics(TCHAR_TO_UTF8(*password), mywordcount);
+
+    output = UTF8_TO_TCHAR(result.c_str());
+    success = true;
+  } catch (const rust::cxxbridge1::Error &e) {
+    success = false;
+    output = TEXT("");
+    output_message = FString::Printf(
+        TEXT("CronosPlayUnreal DevelopmentOnlyGenerateMnemonics Error: %s"),
+        UTF8_TO_TCHAR(e.what()));
+  }
+}
```

Comment:
possibly some notes to be careful about the usage + help to zeroize those strings that aren't copied back to the caller

Reply:
ok

Reply:
working on now

Reply:
added warning, zeroize